spot_ros2_control/hardware/include/spot_ros2_control/spot_joint_map.hpp

```diff
@@ -24,12 +24,23 @@
 
 #include "rclcpp/rclcpp.hpp"
 #include "sensor_msgs/msg/joint_state.hpp"
-#include "std_msgs/msg/float64_multi_array.hpp"
 
 #pragma once
 
 namespace spot_ros2_control {
 
+// Gain values https://github.com/boston-dynamics/spot-cpp-sdk/blob/master/cpp/examples/joint_control/constants.hpp
+// This will be handled via a parameter in the future so there is the option to change them, for now they are hardcoded
+
+// kp and kd gains for a robot without an arm
+const std::vector<float> no_arm_kp = {624, 936, 286, 624, 936, 286, 624, 936, 286, 624, 936, 286};
+const std::vector<float> no_arm_kd = {5.20, 5.20, 2.04, 5.20, 5.20, 2.04, 5.20, 5.20, 2.04, 5.20, 5.20, 2.04};
+// kp and kd gains for a robot with an arm
+const std::vector<float> arm_kp = {624, 936, 286,  624, 936, 286, 624, 936, 286, 624,
+                                   936, 286, 1020, 255, 204, 102, 102, 102, 16.0};
+const std::vector<float> arm_kd = {5.20, 5.20, 2.04, 5.20, 5.20, 2.04, 5.20, 5.20, 2.04, 5.20,
```

Comment:
@khughes-bdai meta, pre-existing: consider using a singleton pattern or getters instead of global variables, they are less susceptible to [static initialization surprises](https://en.cppreference.com/w/cpp/language/siof).